pkg/resources/ratelimit/envoy.go

```diff
@@ -36,15 +36,15 @@ func NewEnvoyProxyServer(ctx context.Context, client k8sclient.Client, logger l.
 	}
 }
 
-func (envoyProxy *envoyProxyServer) CreateEnvoyProxyContainer(dcName, namespace, envoyNodeID, svcProxyName, svcProxyPortName string, svcProxyPort int) (integreatlyv1alpha1.StatusPhase, error) {
+func (envoyProxy *envoyProxyServer) CreateEnvoyProxyContainer(deploymentName, namespace, envoyNodeID, svcProxyName, svcProxyPortName string, svcProxyPort int) (integreatlyv1alpha1.StatusPhase, error) {
 
 	envoyProxy.log.Infof(
 		"Creating envoy sidecar container for: ",
-		l.Fields{"DeploymentConfig": dcName, "Namespace": namespace},
+		l.Fields{"Deployment": deploymentName, "Namespace": namespace},
 	)
 
 	// patches deployment config to add the sidecar container
```

Comment:
Not a big deal but this comment could be updated

```suggestion
	// patches deployment to add the sidecar container
```